.github/workflows/ci.yml

```diff
@@ -9,36 +9,61 @@ on:
       - '**'
 
 jobs:
-  run_tests:
-    name: Tests
-    runs-on: ${{ matrix.os }}
+  run_dbt_schema_builder_tests:
+    name: dbt Schema Builder Tests
+    runs-on: ubuntu-20.04
     strategy:
       matrix:
-        os: [ ubuntu-20.04 ]
-        python-version: [ 3.8 ]
-        toxenv: [ py38,quality,docs ]
-
+        python-version: [3.8, 3.9]
     steps:
-    - name: Checkout Repo
-      uses: actions/checkout@v2
+    - uses: actions/checkout@v1
     - name: setup python
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-
-    - name: Install pip
+    - name: Install pip, setuptools, and wheel
       run: pip install -r requirements/pip.txt
-
     - name: Install dependencies
       run: pip install -r requirements/ci.txt
-
+    - name: Install Tox
+      run: pip install tox tox-gh-actions
     - name: Run Tests
-      env:
-        TOXENV: ${{ matrix.toxenv }}
       run: tox
-
     - name: Run Coverage
-      if: matrix.toxenv=='py38'
       uses: codecov/codecov-action@v2
       with:
         fail_ci_if_error: true
+  run_dbt_schema_builder_code_quality:
+    name: dbt Schema Builder Code Quality
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
Nit: checkout@v2 is out which is apparently slightly faster and other goodies.

Reply:
I updated the version to checkout@v2. Thanks! I'll make sure to keep the branch naming scheme in mind for future PRs.